source/fuzz/transformation_add_parameter.cpp

```diff
@@ -93,9 +99,6 @@ void TransformationAddParameter::Apply(
           return;
         }
 
-        // TODO(https://github.com/KhronosGroup/SPIRV-Tools/issues/3177):
-        //  it would be good to mark this usage of |id| as irrelevant, so that
-        //  we can perform some interesting transformations on it later.
```

Comment:
Is this resolved by a no-op because we deal with it in the fuzzer pass - i.e., we provide irrelevant ids there, and there's thus nothing to do here?  (And indeed it wouldn't be safe here to assume that they are irrelevant?)

Reply:
Yes, this comment was created before we've decided that "irrelevant id" is better than "irrelevant use of id".